guides/source/active_record_validations.md

```diff
@@ -1275,8 +1275,9 @@ irb> error.full_message
 => "Name is too short (minimum is 3 characters)"
 ```
 
-The [`full_message`][] method generates a more user-friendly message, with the capitalized attribute name prepended.
+The [`full_message`][] method generates a more user-friendly message, with the capitalized attribute name prepended. To customize `full_messages`, see [`config.active_model.i18n_customize_full_message`][] config.
```

Comment:
This leaves out the fact that the format can be customized with just `errors.format`, which might mislead some readers.  I think we either need to mention both, or neither, or possibly just link to the I18n guide.

Reply:
I've udpated a paragraph in i18n and linked it from active record validations to cover all the ways to update the format. 

Here is the paragraph:

https://github.com/rails/rails/pull/45470/files#diff-67f722d60df3d44c81af887021a7dcf123bc3ed2b30909b8f8736c43f37722ddR1058